src/panels/lovelace/editor/unused-entities/hui-unused-entities.ts

```diff
@@ -86,54 +75,58 @@ export class HuiUnusedEntities extends LitElement {
             : ""}
         </div>
       </ha-card>
-      <search-input @value-changed=${this._handleSearchChange}></search-input>
       <ha-data-table
-        .headers=${[
-          {
-            name: "Entity",
+        .columns=${{
```

Comment:
You are creating a new object for this on every state call to `render`. Instead we should have this be a static class variable.

```ts
static columns = {

}
```